docs/walkthrough-1.7.md

```diff
@@ -0,0 +1,8 @@
+# Service Catalog Demonstration Walkthrough
+
+This document assumes that you've installed Service Catalog onto your cluster.
+If you haven't, please see [install-1.7.md](./install-1.7.md).
+
+This document is a work in progress. Instructions for the self-guided demo
```

Comment:
I think we should either:

- duplicate the walkthrough content here, and only maintain both until the beta
- expand significantly on the commands to use

Reply:
I already wrote in the 1.6 walkthrough that it's deprecated and may be removed at any time. In a follow-up, I am going to duplicate the existing walkthrough and expand on it. Does that address your concern @pmorie?

Reply:
Yeah, that does, thanks.